src/client/activation/migrateDataScienceSettingsService.ts

```diff
@@ -34,21 +34,29 @@ export class MigrateDataScienceSettingsService implements IExtensionActivationSe
 
     public async activate(resource: Resource): Promise<void> {
         // Only perform the migrate once
-        const migrated = this.persistentStateFactory.createGlobalPersistentState(
+        const migratedSettings = this.persistentStateFactory.createGlobalPersistentState(
             'MigratedDataScienceSettingsService',
             false
         );
-        if (!migrated.value) {
+        const migratedKeybindings = this.persistentStateFactory.createGlobalPersistentState(
+            'MigratedDataScienceKeybindingsService',
+            false
+        );
+        if (!migratedSettings.value) {
             await this.updateSettings(resource);
-            migrated.updateValue(true).ignoreErrors();
+            migratedSettings.updateValue(true).ignoreErrors();
+        }
+        if (!migratedKeybindings.value) {
+            await this.updateKeybindings();
+            migratedKeybindings.updateValue(true).ignoreErrors();
         }
     }
 
     @swallowExceptions('Failed to update settings.json')
     private async fixSettingsFile(filePath: string) {
         let fileContents = await this.fs.readLocalFile(filePath);
 
-        if (!fileContents.toLowerCase().includes('python.datascience')) {
+        if (!fileContents.includes('python.dataScience')) {
```

Comment:
I'm a bit confused here. Didn't we want the case insensitive check? Or was there an issue with that?

Reply:
Yeah I was confused too. Let me try to explain. I went back and looked at our old package.json and our old commands started with all-lowercase `python.datascience`, and our old settings started with camelcased `python.dataScience`. In my first PR (which we bug bashed), keybindings were not being migrated because my code was looking for `python.dataScience` commands, which don't exist. In the second one I made the search case insensitive. This was wrong because the commands are all lowercase. So I don't think we actually want the case insensitive check anymore. Sorry for the multiples on this.

Reply:
Sounds good. Just making sure that I understood.